server/index.js

```diff
@@ -1,12 +1,9 @@
-const express = require('express');
-const reviewsAllRoutes = require('./routes/reviewsAll');
-const reviewCalculationRoutes = require('./routes/reviewCalculationRoutes');
+import express from 'express';
+import reviewsRender from './routes/reviewsRender';
 
 const app = express();
-
-app.use(...reviewsAllRoutes);
-app.use(...reviewCalculationRoutes);
-
 const port = process.env.PORT || 6002;
 
+app.use(...reviewsRender);
```

Comment:
Interesting, so using reviewsRender is now taking care of all prior functionality? Is this essentially serving up your entire frontend as well?

Reply:
Yes, this just inputs all the logic for rendering on the server. I think I will change this from "reviewsRender" to just render when I clone everyone's repos and add their frontends in.